src/components/desktop-exclusive/Sidebar/MainSidebar.tsx

```diff
@@ -1,99 +1,58 @@
 import UserIcon from '@components/common/UserIcon';
 import classNames from 'classnames';
 import Image from 'next/image';
+import MarkdownIt from 'markdown-it';
 
-import logo from 'public/images/logo-mock.bmp';
+import { apiClient } from '@app/apiClient';
 
 type Props = {
   className?: string;
 };
 
-const Sidebar = ({ className }: Props) => {
+const md = new MarkdownIt();
+
+const Sidebar = async ({ className }: Props) => {
+  const formatter = Intl.NumberFormat('en', { notation: 'compact' });
```

Comment:
Is it worth moving `formatter` to a util file in src/utils? I think this could be useful for other counts such as comments, scores (or upvotes/downvotes).

Reply:
Thinking about it now, I don't think this needs to be done for this PR, so we can just ignore the previous comment. We can move this to a util in the future when we work on a component that can make use of this.

Reply:
I moved it 👍